alf/algorithms/taac_algorithm.py

```diff
@@ -411,10 +411,7 @@ def _make_parallel(net):
 
         tau_spec = nest.map_structure(lambda m: action_spec if m else (),
                                       tau_mask)
-        tau_embedding = nest.map_structure(
-            lambda _: torch.nn.Sequential(
-                alf.layers.FC(action_spec.numel, observation_spec.numel)),
-            tau_spec)
+        tau_embedding = nest.map_structure(lambda _: None, tau_spec)
```

Comment:
This change makes the algorithm behave differently. Does it change the experiment results?
And if all None, there is no need to set input_preprocessors below.

Reply:
Just in case, I've reverted this change to be consistent with the experiments in the paper.